### DISCUSSION TITLE
Issue with Agile Prices for 2nd August on App/HA

### DISCUSSION THREAD

**Discussion by anothermort** [BottlecapDave/HomeAssistant-OctopusEnergy#337] [General]
This probably an API issue but thought others might need to know on here.

Agile prices for 2nd August seem to be duplicating on Octopus App
and
BottlecapDaves agile price entity in HA is broken for agile price rates for today and tomorrow!

So if you need it to automate anything in HA might be worth checking you setup.

**Comment by gorstj** [comment]
I am seeing that the Agile data in HA isn't working as of 4pm today.

Was just about to start searching in debug logs thinking I had broken it..... but seems like an Octopus end issue
